nextcord/types/interactions.py

```diff
@@ -228,6 +228,13 @@ class Interaction(_InteractionOptional):
     version: int
 
 
+class MessageInteraction(TypedDict):
```

Comment:
`member` can be included as a non-required key

https://discord.com/developers/docs/interactions/receiving-and-responding#message-interaction-object-message-interaction-structure